frontend/app/components/wp-watchers/wp-watcher-entry.directive.html

```diff
@@ -0,0 +1,34 @@
+<span class="inplace-edit--read watcher-element"
+      ng-focus="$ctrl.focus()"
+      ng-blur="$ctrl.blur()">
+  <span class="inplace-editing--trigger-container">
+    <span class="inplace-editing--trigger-link"
+          data-ng-class="{'-focus': $ctrl.state.focussing}">
+      <span class="inplace-editing--container">
+        <span class="inplace-edit--read-value" >
+          <span data-ng-class="{'deleting': $ctrl.state.deleting}">
+            <a ng-href="{{ $ctrl.watcher.showUser.href }}"
+               title="{{ $ctrl.watcher.name }}">
+              <img ng-attr-src="{{ $ctrl.watcher.avatar }}"
+                   ng-attr-alt="{{ $ctrl.watcher.name }}"
+                   ng-show="$ctrl.watcher.avatar"
+                   class="avatar-mini">
+              <span class="work-package--watcher-name">{{ $ctrl.watcher.name }}</span>
+            </a>
+          </span>
+        </span>
+        <accessible-by-keyboard execute="$ctrl.remove()"
+                                aria-label="{{ ::$ctrl.text.remove }}"
+                                link-title="{{ ::$ctrl.text.remove }}"
+                                link-class="inplace-edit--icon-wrapper remove-watcher-btn">
+          <icon-wrapper icon-name="remove"
+                        data-icon-title="{{::$ctrl.text.remove}}">
+          </icon-wrapper>
+        </accessible-by-keyboard>
+      </span>
+    </span>
+  </span>
+</span>
```

Comment:
The added watchers are no longer removable via keyboard as the delete button is no longer displayed when focussing the link:

Formerly:
![image](https://cloud.githubusercontent.com/assets/617519/20308732/8c5c60e2-ab45-11e6-8fdb-56652744fda4.png)
Now:
![image](https://cloud.githubusercontent.com/assets/617519/20308780/ba346c62-ab45-11e6-800e-241454dbc2a9.png)